docs/src/main/sphinx/develop/client-protocol.rst

```diff
@@ -27,8 +27,11 @@ only required on the initial ``POST`` request, and not when following the
 ``nextUri`` links.
 
 If the client request returns an HTTP 503, that means the server was busy,
```

Comment:
Trino actually doesn't return 503. That was the intended code for gateways, etc., but in practice, we need to support 502, 503, 504, so we can just mention all three here.

Reply:
reworded docuementation.